pkg/tsdb/cloudwatch/cloudwatch_test.go

```diff
@@ -327,6 +327,94 @@ func TestQuery_ResourceRequest_DescribeAllLogGroups(t *testing.T) {
 		assert.Equal(t, stringsToSuggestData([]string{
 			"group_a", "group_b", "group_c", "group_x", "group_y", "group_z",
 		}), suggestDataResponse)
+
+		firstCall := cli.calls.describeLogGroups[0]
+		secondCall := cli.calls.describeLogGroups[1]
+
+		expectedFirst := &cloudwatchlogs.DescribeLogGroupsInput{
+			Limit: aws.Int64(50),
+		}
+		expectedSecond := &cloudwatchlogs.DescribeLogGroupsInput{
+			Limit:     aws.Int64(50),
+			NextToken: aws.String("foo"),
+		}
+		assert.Equal(t, expectedFirst, firstCall)
+		assert.Equal(t, expectedSecond, secondCall)
```

Comment:
```suggestion
```

I think we don't need these assertions, what do you think? Apologies if I'm missing something that you wanted to show here.;

Reply:
This test case tries to fetch multiple "pages", so I thought it makes sense to assert that the "NextToken" from the first response is passed in the second request. You're right in that this modification doesn't really fit for what this PR is trying to achieve.

Reply:
If that's ok with you, let's leave that out for now and merge in everything else you have here.

Reply:
Sure!